src/components/Fields/RequiredIconAndTooltip.scss

```diff
@@ -0,0 +1,9 @@
+.uir-textfield-inner,
+.uir-textarea-inner {
+
+    & > .icon-required-tooltip {
+        position: absolute;
+        top: 20px;
+        width: auto !important;
```

Comment:
Could you please add a comment in code explaining why we need to use `!important`?
Generally we should avoid it unless there's a super good reason to keep it.

Reply:
I know we should avoid, I've used it here because the `icon-required-tooltip' clashes with the standard TextField or TextArea tooltip... I'll see if I can refactor to remove and if not I'll add a comment

Reply:
Found a better solution by adjusting the CSS classes 👍